yarn-project/end-to-end/src/shared/uniswap_l1_l2.ts

```diff
@@ -940,7 +940,7 @@ export const uniswapL1L2TestSuite = (
         uniswapPortal.simulate.swapPublic(swapArgs, {
           account: ownerEthAddress.toString(),
         } as any),
-      ).rejects.toThrow('The contract function "swapPublic" reverted.');
```

Comment:
I think this is taken as a substring, the only difference with the new regex is the trailing "."